.gitignore

```diff
@@ -7,3 +7,20 @@ dist
 /*.prql
 */*.prql
 target
+playground/node_modules
```

Comment:
`node_modules` is in https://github.com/prql/prql/blob/main/playground/.gitignore#L4 — is there a reason we need it here too?